.travis.yml

```diff
@@ -10,19 +10,7 @@
 language: php
 
 php:
-    - 5.4
```

Comment:
is just to focus :+1: speed up travis and count what matters the most for this branch